### DISCUSSION TITLE
Student verification approved but nothing changed yet ...

### DISCUSSION THREAD

**Discussion by noobyzy** [community/community#66886] [GitHub Education] [answered]
### Hi everyone,

I submitted my verification materials for the student pack 2 days ago. It's been approved immediately, see screenshot below
![image](https://github.com/community/community/assets/42728483/6d4dc3bd-3494-40f3-a37a-4a2911f8c9b4)
But in settings/Copilot, I still cannot get student access for copilot see screenshot below
![image](https://github.com/community/community/assets/42728483/c75845b3-3c27-46dc-a5a9-1b67e2d39e04)
And I assume there should be an email to inform me? And I received none yet. 
Please help with the issue.

**Comment by arafajs** [comment, marked as answer]
According to the GitHub documentation, the activation process of Benefits for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by noobyzy** [reply]
Thanks for the reply
